docs/feature_os_detection.md

```diff
@@ -32,6 +32,54 @@ enum {
 ?> Note that it takes some time after firmware is booted to detect the OS.
 This time is quite short, probably hundreds of milliseconds, but this data may be not ready in keyboard and layout setup functions which run very early during firmware startup.
 
+## Callbacks :id=callbacks
+
+If you want to perform custom actions when the OS is detected, then you can use the `process_detected_host_os_kb` function on the keyboard level source file, or `process_detected_host_os_user` function in the user `keymap.c`.
+
+```c
+bool process_detected_host_os_kb(os_variant_t os) {
+    if (!process_detected_host_os_user(os)) {
+        return false;
+    }
+    switch (os) {
```

Comment:
This is more a nit, but `os` seems too short and unclear of a name here.  It's kind of hard to read in the code, and not exactly clear. 


Eg, this feels a lot clearer and more readable to me: 
```suggestion
bool process_detected_host_os_kb(os_variant_t detected_os) {
    if (!process_detected_host_os_user(detected_os)) {
        return false;
    }
    switch (detected_os) {
```

Reply:
Done as well